### PR TITLE
New Thrasher – FGM Thrasher

### PR DESCRIPTION
**Pull request by sammorrisdesign**
fgm-thrasher

### CODE REVIEW

File: embeds/fgm-campaign/style.scss [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        float: right;
+        width: 50%;
+    }
+}
+
+.fgm-campaign__title,
+.fgm-campaign__description {
+    margin: 0;
+    padding: 0;
+    font-family: "Guardian Egyptian Web", "Guardian Text Egyptian Web", Georgia, serif;
```
Comment (sndrs) [inline embeds/fgm-campaign/style.scss]: shouldn't these use the guardian typography mixins?

File: embeds/fgm-campaign/style.scss [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    line-height: 1;
+    color: white;
```
Comment (sndrs) [inline embeds/fgm-campaign/style.scss]: use hex

File: embeds/fgm-campaign/style.scss [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+    desktop:         980px,
+    leftCol:         1140px,
+    wide:            1300px
+);
+
+@import 'shared/_imports';
+
+$c-red: #951c55;
+
+@include base(fgm-campaign, mix(#7d7569, #fff, 20%));
```
Comment (sndrs) [inline embeds/fgm-campaign/style.scss]: six char numbers

File: embeds/fgm-campaign/style.scss [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    @include mq($until: mobileLandscape) {
+        margin-top: 40%;
+    }
+
+    @include mq(mobileLandscape, tablet) {
+        margin-top: 150px;
+    }
+
+    @include mq($until: 1024px) {
+        background-color: rgba(#7d7569, 0.5);
```
Comment (sndrs) [inline embeds/fgm-campaign/style.scss]: no leading 0

File: embeds/fgm-campaign/style.scss [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+    color: white;
+    margin-top: 20px;
+
+    @include mq(tablet) {
+        position: absolute;
+        bottom: 10px;
+    }
+
+    &:hover {
+        background-color: mix($c-red, #000, 90%);
```
Comment (sndrs) [inline embeds/fgm-campaign/style.scss]: nice

**Comment by sndrs** [comment]
looks like the scss needs the standard linting tasks adding, as well as the site-wide mixins. you should be able to use bower to automate that if necessary.

**Comment by sndrs** [comment]
does this prevent the video from being downloaded below desktop?

**Comment by sammorrisdesign** [comment]
Linting & mixins: Yeah there's a case to be made for these, especially the ones that affect the output. I'll make sure to look into it soon.

Autoplaying videos don't autoplay on mobile. Most mobile browsers force the user to interact before fetching data.

File: embeds/fgm-campaign/style.scss (hunk 1 above)
Comment (sndrs) [inline embeds/fgm-campaign/style.scss]: not going to use these?

**Comment by sndrs** [comment]
this should really use guss/pasteup font mixins and include linting in the build process. before it's ever used again the video needs cache headers adding, and it needs confirmation that the video doesn't download when it's not viewed.

could you just keep this as branch for future ref?

**Comment by sammorrisdesign** [comment]
For video, autoplaying being disabled on mobile seems to be a standard browser pattern. It's disabled by default for bandwidth reasons on both [iOS](http://stackoverflow.com/questions/12496144/can-you-autoplay-html5-videos-on-the-ipad) and [Android](https://code.google.com/p/chromium/issues/detail?id=159336).

**Comment by sammorrisdesign** [comment]
Updated to use frontend components... going to merge in now.

**Pull request merged**